packages/victory-pie/src/victory-pie.js

```diff
@@ -89,6 +89,7 @@ class VictoryPie extends React.Component {
     cornerRadius: PropTypes.oneOfType([CustomPropTypes.nonNegative, PropTypes.func]),
     data: PropTypes.array,
     dataComponent: PropTypes.element,
+    disableInlineStyes: PropTypes.bool,
```

Comment:
I think for this work to be complete for `VictoryPie`, you'll also need to pass the `disableInlineStyles` prop through to `Slice` component via data props here: https://github.com/FormidableLabs/victory/blob/main/packages/victory-pie/src/helper-methods.js#L249

You could also save a bit of work here: https://github.com/FormidableLabs/victory/blob/main/packages/victory-pie/src/helper-methods.js#L259

Reply:
I think passing through the `disableInlineStyles` prop for `VictoryLabel` would be taken care of in `LabelHelpers.getProps`

Reply:
Done!